src/vpic/vpic.h

```diff
@@ -652,6 +646,86 @@ class vpic_simulation {
      return find_species_id( id, species_list );
   }
 
+  // enum class Tracertype { Copy, Move };
+  #ifdef VPIC_GLOBAL_PARTICLE_ID
+  // REVIEW: make the name a std::string? or match define_species?
```

Comment:
I'm flexible, but I think we go string unless it causes some problems? I'd like to get away from all the `char*` stuff eventually

Reply:
std::string it is then.